hapistrano.cabal

```diff
@@ -59,15 +59,15 @@ test-suite hapistrano-test
   hs-source-dirs:      spec
   main-is:             Spec.hs
   other-modules:       System.HapistranoSpec
-  build-depends:       base
+  build-depends:       base >= 4.5 && < 4.10
                      , hapistrano
-                     , directory
-                     , either
-                     , filepath
-                     , hspec
-                     , mtl
-                     , process
-                     , temporary
+                     , directory >= 1.2.1 && < 1.3
+                     , either >= 4.0 && < 4.6
+                     , filepath >= 1.3 && < 1.5
+                     , hspec > 2.0 && < 2.5
+                     , mtl >= 2.0 && < 3.0
+                     , process >= 1.2 && < 1.5
+                     , temporary >= 1.2 && < 1.5
```

Comment:
Same here. We don't need version constraints for base, directory, filepath, and process.